core/tests/puppeteer-acceptance-tests/user-utilities/super-admin-utils.ts

```diff
@@ -39,12 +39,16 @@ export class SuperAdmin extends BaseUser {
     const allRoles = await this.page.$$('.mat-option-text');
     for (let i = 0; i < allRoles.length; i++) {
       const roleText = await this.page.evaluate(
-        (role: HTMLElement) => role.innerText,
+        (element: HTMLElement) => element.innerText,
         allRoles[i]
       );
       if (roleText.toLowerCase() === role) {
-        await allRoles[i].click();
-        await this.page.waitForNetworkIdle();
+        // Here we are clicking on the role element in the DOM since the options
+        // might not be in view.
+        await Promise.all([
+          allRoles[i].evaluate(element => (element as HTMLElement).click()),
```

Comment:
I don't really think we need the comment here, element is a Element which is not an HTMLElement, HTMLElement is built up on top of Element which includes the click. Here I am casting because the element is 100% a HTMLElement don't need to do something like element insance of HTMLElement. Here I am just clicking in the DOM as in evaluating instead of using the ElementHandle.click().

Reply:
Here I changed the code because before I was using clickOn or this.page.click which uses the mouse to click but we can't move the option into the viewport which resulted in options like Question Admin being mistaken for Moderator (so here we just use the in built DOM click() to click regardless if the option is in view or not).